proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -196,4 +196,7 @@ message Telemetry {
     // Whether to capture metrics or not. The metrics will be otel go metrics.
     // See https://github.com/open-telemetry/opentelemetry-go/tree/main/metric
     google.protobuf.BoolValue metrics_enabled = 2;
+
```

Comment:
lets move this and the above enabled flag to a common metrics message and deprecate the metrics_enabled one